### PR TITLE
Add Morph and needed packages

### PR DESCRIPTION
**Pull request by ulrikstrid**

### CODE REVIEW

File: ocaml/default.nix [hunk 1]
```diff
@@ -3,7 +3,6 @@
 oself: osuper:
```
Comment (ulrikstrid) [inline ocaml/default.nix:3]: Sorry about the huge diff here, I installed a nix formatter and forgot to check the diff here, I can fix that.

File: ocaml/subscriptions-transport-ws/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      useDune2 = true;
+      inherit src;
+
+      propagatedBuildInputs = with ocamlPackages; [
+        websocketaf
+        graphql
+      ];
+
+      meta = {
+        license = stdenv.lib.licenses.mit;
```
Comment (anmonteiro) [inline ocaml/subscriptions-transport-ws/default.nix:24]:
```suggestion
        license = stdenv.lib.licenses.bsd3;
```

File: ocaml/default.nix (hunk 1 above)
Comment (anmonteiro) [reply ocaml/default.nix:3]: please revert this diff. happy to merge a PR that formats the whole thing, separately, if you want.

File: ocaml/default.nix [hunk 3]
```diff
@@ -71,6 +71,10 @@ let
       ocamlPackages = oself;
     };
 
+  morphPackages = callPackage ./morph {
```
Comment (anmonteiro) [inline ocaml/default.nix:74]: you need to `//` them below

Comment (anmonteiro) [reply ocaml/default.nix:74]: I think you also forgot to include session and cookie now

Comment (ulrikstrid) [reply ocaml/default.nix:78]: Should be fixed now, my git-fu is not as strong as it should be so I reverted a bit too much 😅

**Pull request merged**